src/ngraph/runtime/backend.cpp

```diff
@@ -39,12 +40,41 @@ runtime::Backend::~Backend()
 {
 }
 
+void* runtime::Backend::open_shared_library(const string& type)
+{
+    void* handle = nullptr;
+    vector<string> names;
+    names.push_back("lib" + type + ".so");
```

Comment:
Why do we need all this DWIM for names? Can't we just define what the name needs to look like and enforce it?

Reply:
Sure. What should we define it to be?